web-animations-1/Overview.bs

```diff
@@ -822,6 +831,29 @@ in which they are created. Certain <a lt="animation class">classes of
 animations</a>, however, may provide alternative means of ordering animations
 (see [[#animation-classes]]).
 
+To convert a <dfn lt="time-based animation to a proportional animation">,
+time-based animation to a proportional animation</dfn>, effectively the
+[=timeline duration=] is divided up by the relative proportions of the
+specified times of the <a>animation</a>, <var>animation</var>. Concretely:
+
+1.  If the [=iteration duration=] is auto, then perform the following steps.
+
+    1.   Compute [=start delay=] and [=end delay=] to 0, as it is not possible
+         to mix time and proportions. Future versions may allow these properties
+         to be assigned percentages.
+
```

Comment:
Need "otherwise" close? The rest of the bullets are applied when iteration duration is not "auto".

Reply:
Thanks, done.